.github/workflows/release.yml

```diff
@@ -0,0 +1,23 @@
+name: Release Charts
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
```

Comment:
Does checkout@v2 work without the following unshallow?
```yaml
      - name: Fetch history
        run: git fetch --prune --unshallow
```

Reply:
why do we need `--unshallow`?

Reply:
checkout@v1 used to do shallow clones i'm not sure if the releaser needs an shallow clone or not. I guess the only way to find out if it works without is by merging this. I've only used v2 with the described unshallow step but never tested if it was really needed.